deploy/29_faucet.ts

```diff
@@ -101,6 +103,28 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
         });
         await tx.wait();
         hre.deployments.log('Updated HLG reference');
+        hre.deployments.log('Transferring 1M HLG to faucet');
+        const transferTx = await hlgContract.transfer(
+          futureFaucetAddress,
+          BigNumber.from('1' + '000' + '000' + '000000000000000000'),
+          {
+            ...(await txParams({
```

Comment:
So for `txParams` and `hre.ethers.provider.estimateGas` I think it would be clearer if those were put into variables and then passed into the `hlgContract.transfer` function